package.json

```diff
@@ -13,6 +13,7 @@
     "lint:html": "htmlhint --config ./tests/linters/.htmlhintrc ./dist/app/components/**/*.html",
     "backstop:ref": "backstop --config=tests/backstop/backstop.js reference --docker",
     "backstop:test": "backstop --config=tests/backstop/backstop.js test --docker",
+    "backstop:approve": "backstop --config=tests/backstop/backstop.js approve --docker",
```

Comment:
The `--docker` flag isn't doing anything here

Reply:
Good spot, I've removed the `--docker` flag.